tuf/repository_tool.py

```diff
@@ -2506,89 +2502,47 @@ def delegate_hashed_bins(self, list_of_targets, keys_of_hashed_bins,
     logger.info(repr(len(list_of_targets)) + ' total targets.')
```

Comment:
Nitpick: Could these separate `info` calls be merged into a single message? That would eliminate the chance of them being spread when integrated into a logging system with multiple parallel sources.

Reply:
Good idea, I've pushed that in a follow-on patch